common/src/main/java/org/mcaccess/minecraftaccess/utils/NarrationUtils.java

```diff
@@ -69,26 +68,61 @@ public static String narrateEntity(Entity entity) {
 
         List<String> equipments = new ArrayList<>();
 
-        if (entity instanceof AnimalEntity animalEntity) {
-            switch (animalEntity) {
-                case SheepEntity sheepEntity -> text = getSheepInfo(sheepEntity, text);
-                case TameableEntity tameableEntity -> {
-                    // wolf, cat, parrot
-                    String isTamedText = I18n.translate("minecraft_access.read_crosshair.is_tamed", text);
-                    text = tameableEntity.isTamed() ? isTamedText : text;
-                    text = tameableEntity.isInSittingPose() ? addSittingInfo(text) : text;
+        if (ReadCrosshairConfigMap.getInstance().isSpeakAdditionalEntityPoses()) {
+            switch (entity.getPose()) {
+                case SLEEPING -> text = I18n.translate("minecraft_access.read_crosshair.sleeping", text);
+                case DYING -> text = I18n.translate("minecraft_access.read_crosshair.dying", text);
+                case DIGGING -> text = I18n.translate("minecraft_access.read_crosshair.digging", text);
+                case GLIDING -> text = I18n.translate("minecraft_access.read_crosshair.gliding", text);
+                case ROARING -> text = I18n.translate("minecraft_access.read_crosshair.roaring", text);
+                case SLIDING -> text = I18n.translate("minecraft_access.read_crosshair.sliding", text);
+                case SWIMMING -> text = I18n.translate("minecraft_access.read_crosshair.swimming", text);
+                case SITTING -> entityIsSitting = true;
+                case CROAKING -> text = I18n.translate("minecraft_access.read_crosshair.croaking", text);
+                case EMERGING -> text = I18n.translate("minecraft_access.read_crosshair.emerging", text);
+                case SHOOTING -> text = I18n.translate("minecraft_access.read_crosshair.shooting", text);
+                case INHALING -> text = I18n.translate("minecraft_access.read_crosshair.inhaling", text);
+                case SNIFFING -> text = I18n.translate("minecraft_access.read_crosshair.sniffing", text);
+                case CROUCHING -> text = I18n.translate("minecraft_access.read_crosshair.crouching", text);
+                case LONG_JUMPING -> text = I18n.translate("minecraft_access.read_crosshair.long_jumping", text);
+                case USING_TONGUE -> text = I18n.translate("minecraft_access.read_crosshair.using_tongue", text);
+                case STANDING -> {
                 }
-                case FoxEntity foxEntity -> text = foxEntity.isSitting() ? addSittingInfo(text) : text;
-                case PandaEntity pandaEntity -> text = pandaEntity.isSitting() ? addSittingInfo(text) : text;
-                case CamelEntity camelEntity -> text = camelEntity.isSitting() ? addSittingInfo(text) : text;
                 default -> {
+                    log.warn("Unhandled pose found: " + entity.getPose().name() + " for additional pose narration in Narration Utils");
```

Comment:
Please use proper string interpolation
```suggestion
                    log.warn("Unhandled pose found: {} for additional pose narration in Narration Utils", entity.getPose().name());
```